### PR TITLE
docs: Restructure promotion docs

### PR DESCRIPTION
**Pull request by yiannistri**
**What changed?**
Updated pipeline documentation.

**Why was this change made?**
To reflect the latest changes.

**How was this change implemented?**
N/A

**How did you validate the change?**
N/A

**Release notes**
N/A

**Documentation Changes**
N/A

### CODE REVIEW

**Comment by makkes** [comment]
[Preview](https://staging.docs.gitops.weave.works/update-promotion-docs/docs/next/enterprise/pipelines/promoting-applications/)

File: website/docs/enterprise/pipelines/promoting-applications.mdx [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+Before configuring any of the above promotion strategies, you need to setup notifications from all your environments so that whenever a new version gets deployed, the promotion webhook component of the pipeline controller is notified and takes an action based on the pipeline definition. The rest of this guide describes the configuration needed to setup application promotion via pipelines.
+
+## Prerequisites
+
+Applications deployed in leaf clusters use the Flux notification controller running on each leaf cluster, to notify the management cluster of a successful promotion. This requires network connectivity to be established between the leaf cluster and the management cluster.
+
+The component responsible for listening to incoming notifications from leaf clusters is the pipeline controller. It hosts a webhook service that needs to be exposed via an ingress to make it available for external calls. This service/ingress is configured via the Weave GitOps Enterprise Helm chart manifest, an example of which is shown below using the NGINX ingress controller:
```
[the remark this answers is not in the document]
Comment (yiannistri) [reply website/docs/enterprise/pipelines/promoting-applications.mdx:38]: 👍  I'll change the wording to make this more explicit.

File: website/docs/enterprise/pipelines/promoting-applications.mdx [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+  values:
+    enablePipelines: true
+    pipeline-controller:
+      promotion:
+        ingress:
+          enabled: true
+          className: nginx
+          annotations:
+            cert-manager.io/cluster-issuer: letsencrypt
+            nginx.ingress.kubernetes.io/rewrite-target: /$1
```
[the remark this answers is not in the document]
Comment (yiannistri) [reply website/docs/enterprise/pipelines/promoting-applications.mdx:51]: I have removed the rewrite rule and added a section to explain this better.

**Review by makkes — APPROVED** [review]
looks great

**Pull request merged**